.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   },
   ignorePatterns: ['.eslintrc.js'],
   rules: {
+    '@typescript-eslint/no-floating-promises': 1,
```

Comment:
it is irrelevant but I am curious. Should we use `on` here? Is this even available - I have no clue of eslint config. But seeing `off` being used below. I thought this would make it consistent.

Reply:
That's what I also thought ... then encountered this 😆 
<img width="723" alt="image" src="https://user-images.githubusercontent.com/85513542/203762382-218d49cd-8d2e-405f-92ee-c0924c3974fc.png">

Reply:
Better? 😉 
<img width="518" alt="image" src="https://user-images.githubusercontent.com/85513542/203762556-50bad49b-f009-4c2d-a379-7e557cbbaa13.png">

Reply:
💪